metaflow/datatools/s3op.py

```diff
@@ -159,7 +159,7 @@ def op_info(url):
                                 raise RuntimeError("Could not load file")
                         tmp.close()
                         os.rename(tmp.name, url.local)
-                    except client_error as err:
+                    except s3client.error as err:
```

Comment:
I left a comment about this in another PR too: `.error` is a side-effectful property which is quite confusing in this context